setup.cfg

```diff
@@ -72,21 +106,7 @@ test =
 test-swift =
     python-swiftclient
 
-[global]
-setup-hooks =
-    pbr.hooks.setup_hook
-
-[build_py]
-pre-hook.build_config = gnocchi.genconfig.prehook
```

Comment:
Now the final tarball does not have `gnocchi.conf`. Not sure it's a big deal since we have a default config generator. Just saying.

Reply:
I have put it back.